### PR TITLE
Update Import/Export project.

### PR DESCRIPTION
**Pull request by robinsh**
Rewrote the Import/Export project per requests from Cristian Pop.

### CODE REVIEW

File: iot-hub/Samples/service/ImportExportDevicesSample/ImportExportDevicesSample.csproj [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Devices" Version="1.27.0-preview-*" />
```
Comment (CIPop) [inline iot-hub/Samples/service/ImportExportDevicesSample/ImportExportDevicesSample.csproj]: Should be Version="1.*"

Comment (robinsh) [reply iot-hub/Samples/service/ImportExportDevicesSample/ImportExportDevicesSample.csproj]: The others aren't 1.*…

**Comment by robinsh** [comment]
removed the imp/exp project under device.

File: iot-hub/Samples/service/ImportExportDevicesSample/ImportExportDevicesSample.csproj (hunk 1 above)
Comment (robinsh) [reply iot-hub/Samples/service/ImportExportDevicesSample/ImportExportDevicesSample.csproj]: In fact, CleanUpDevicesSample reference version 1.17 of that assembly, which is old. The last 3 are ok. I'll fix mine.

**Pull request merged**